samples/TagHelperSample.Web/wwwroot/original.js

```diff
@@ -0,0 +1,7 @@
+﻿function foo() {
```

Comment:
we probably need them too, but not going to bother with spending lots of time on this. We just add them whenever we make a change.